l1-contracts/src/core/Rollup.sol

```diff
@@ -149,12 +150,19 @@ contract Rollup is IRollup {
       revert Errors.Rollup__InvalidInHash(inHash, header.contentCommitment.inHash);
     }
 
-    // Currently trying out storing each tx's L2 to L1 messages in variable height trees (smallest tree required)
-    // => path lengths will differ and we cannot provide one here
-    // We can provide a minimum which is the height of the rollup layers (txTreeHeight) and the smallest 'tree' (1 layer)
-    uint256 l2ToL1TreeMinHeight = header.contentCommitment.txTreeHeight + 1;
+    // Both rollup tx trees and each tx's L2 to L1 messages are variable height.
+    // TODO: Is the below necessary? Is it fine to not supply any height?
+    // Min size = smallest path of the rollup tree + 1
+    // Max size = largest path of the rollup tree + log2(max_messages)
+    (uint256 min, uint256 max) = MerkleLib.computeMinMaxPathLength(header.contentCommitment.numTxs);
```

Comment:
I constrain the path length in the outbox to be between min and max, but am unsure if this is really required at all.
To consume a message, a user must provide a preimage, not an already hashed node, so they cannot provide a shortened path as an attack without finding a collision. Perhaps no constraint on the path length is needed?

Reply:
Added #7218 - will eventually remove this logic anyway to have a fixed length path. Added comments to clarify and reverted adding max length